src/set.rs

```diff
@@ -11,16 +11,25 @@ use serde::{Deserialize, Serialize};
     PartialEq,
     Eq,
     PartialOrd,
-    AtatLen,
     Ord,
     hash32_derive::Hash32,
     Default,
     Serialize,
     Deserialize,
 )]
-#[cfg_attr(feature = "defmt", derive(defmt::Format))]
 pub struct Handle(pub u8);
 
+#[cfg(feature = "defmt")]
+impl defmt::Format for Handle {
+    fn format(&self, f: defmt::Formatter) {
+        self.0.format(f)
+    }
+}
+
+impl atat::AtatLen for Handle {
+    const LEN: usize = 3;
```

Comment:
I think this will give exactly the same as the derive?

If anything it would make sense to change it to 1, as i don't think there are any ublox devices that can handle more than 10 sockets at a time? (0-9)

Reply:
Yes, for sure, it is the same as the derive and the 1 was my original intention :D

The defmt implementation is a left over because I haven't pushed yesterday evening...

Reply:
This looks okay to me. 
I don't have strong feelings on whether the defmt impl should render the `Handle(X)` or just `X`, so if you do, that is fine by me :)

Reply:
Removed it from the last commit. It should be fine now.
This combined with the ublox-cellular changes gets it building for me with the sara-r5 + defmt features